x-pack/plugins/lens/public/visualizations/metric/constants.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const LENS_METRIC_ID = 'lnsMetricNew'; // TODO - rename old one to "legacy"
```

Comment:
Is it worth a migration to rename the old metric visualization to `lnsLegacyMetric`?

Reply:
Let's wait for GA with that